### PR TITLE
Replaced favorite word/icon with star one

### LINKED ISSUE #2041: consistency between favorite / star(red) terms
### Issue details
There is some inconsistency in the interface for favorite articles. Sometimes you'll find the term star(red) like on the sidebar but sometimes you'll find the term favorite. On the card of articles the icon to make the article favorite is an heart and not a star :). I know I'm nit-picking

### Environment
This is not relevant for this bug but anyway I provide it
* wallabag version : 2.0.4
* How did you install wallabag? git clone
* Last wallabag version that did not exhibit the issue (if applicable): N/a
* php version: 7.0
* OS: ubuntu 16.04
* type of hosting: dedicated
* which storage system you choose at install: SQLite

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #2041 
| License       | MIT

Fix #2041

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ entry:
         reading_time_less_one_minute: 'estimated reading time: <small class="inferieur">&lt;</small> 1 min'
         original_article: 'original'
         toogle_as_read: 'Toggle mark as read'
-        toogle_as_star: 'Toggle favorite'
+        set_as_star: 'Toggle star'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: :-1:

**Comment by tcitworld** [comment]
Maybe `set_as_starred` instead of `set_as_star`

**Comment by nicosomb** [comment]
@tcitworld I changed it.

File: src/Wallabag/CoreBundle/Controller/EntryController.php [hunk 2]
```diff
@@ -357,7 +357,7 @@ public function toggleArchiveAction(Request $request, Entry $entry)
     }
 
     /**
-     * Changes favorite status for an entry.
+     * Changes star status for an entry.
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Controller/EntryController.php]: starred

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 3]
```diff
@@ -146,7 +146,7 @@ entry:
         reading_time_less_one_minute: 'estimated reading time: <small class="inferieur">&lt;</small> 1 min'
         original_article: 'original'
         toogle_as_read: 'Toggle mark as read'
-        toogle_as_star: 'Toggle favorite'
+        toogle_as_star: 'Toggle star'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: Toggle starred

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 4]
```diff
@@ -176,7 +176,7 @@ entry:
             back_to_homepage: 'Back'
             set_as_read: 'Mark as read'
             set_as_unread: 'Mark as unread'
-            set_as_favorite: 'Favorite'
+            set_as_starred: 'Toggle star'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: Same

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig [hunk 5]
```diff
@@ -72,7 +72,7 @@
                 $("body").css("cursor", "auto");
             });*/
 
-            // toggle favorite property of current article
+            // toggle star property of current article
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entry.html.twig]: starred

**Comment by tcitworld** [comment]
There's a few more changes to make. :-)

**Comment by tcitworld** [comment]
:+1:

**Pull request merged**